Manager 4.3: LTS (#4109)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/lts.adoc

[[server-lts]]
= Enable LTS

> Long Term Service Pack (LTS) extends the lifecycle of {productname}.

Reviewer: ```suggestion
Long Term Support (LTS) extends the lifecycle of {productname}.
```

It is available as an extension for {productname} Server, Proxy, and Retail Branch Server.

> For more information about LTSS in general, refer to https://www.suse.com/products/long-term-service-pack-support/

Reviewer: ```suggestion
For more information about LTS in general, refer to https://www.suse.com/products/long-term-service-pack-support/
```

> [[server-lts]]

Author: ```suggestion
[[lts]]
```
